src/main/java/io/vertx/core/file/impl/FileResolverImpl.java

```diff
@@ -49,6 +49,7 @@ public class FileResolverImpl implements FileResolver {
   public static final String CACHE_DIR_BASE_PROP_NAME = "vertx.cacheDirBase";
   private static final boolean NON_UNIX_FILE_SEP = File.separatorChar != '/';
 
+  private static final String JAR_URL_SEP = "!/";
```

Comment:
move this with the group of static final above instead of the class fields

Reply:
Done by moving the line up.